api/DOMRectReadOnly.json

```diff
@@ -74,7 +74,7 @@
               "version_added": "79"
             },
             "firefox": {
-              "version_added": true
+              "version_added": "62"
```

Comment:
Why is only the constructor updated here? I think it'd be better to split out the Firefox changes for geometry APIs into its own PR for review by @ddbeck.

Reply:
It's probably because the APIs themselves were implemented in Firefox 32 or lower (perhaps 31 if I had to guess), which we don't have data for in the collector.  I can fix that, though, run a few collectors on older Firefox versions.

Reply:
I see. I've confirmed this change with https://software.hixie.ch/utilities/js/live-dom-viewer/saved/8398 in Firefox 61+62.

Reply:
I see. I've confirmed this change with https://software.hixie.ch/utilities/js/live-dom-viewer/saved/8398 in Firefox 61+62.